pom.xml

```diff
@@ -613,6 +613,12 @@
                 <artifactId>kubernetes-client</artifactId>
                 <version>${kubernetes-client.version}</version>
             </dependency>
+            <!-- according to https://github.com/fabric8io/kubernetes-client/blob/master/doc/MIGRATION-v6.md#okhttp-httpclient -->
```

Comment:
how about explain in this way: `According something, then we should do something balalah or maybe to avoid something`